src/views/CollectionIndex/Page.tsx

```diff
@@ -0,0 +1,63 @@
+import topArrow from "images/arrow-down.svg";
+import bottomArrow from "images/arrow-up.svg";
+import React, { useState } from "react";
+import { Link } from 'react-router-dom';
+import { PageHeader } from "../../components/Header/PageHeader";
+
+
+const Page = props => {
+  const [show, showOverlay] = useState(false);
+  const { data, history } = props;
+  const toShow = localStorage.getItem('show')
+  if(toShow === "true") {
+    showOverlay(true)
+  }
+  localStorage.removeItem('show')
+
+  const handleOverlay = () => {
+    showOverlay(!show);
+  };
+
+  const handleClick = () => {
+    history.push("/collections/cabinets");
+  };
```

Comment:
`handleClick` is confusing. Because there's multiple actions in the header. Perhaps `handleBack` is better.